tests/test_filelock.py

```diff
@@ -576,3 +576,20 @@ def mess_with_file() -> None:
             results.append(executor.submit(mess_with_file))
 
     assert all(r.result() is None for r in results)
+
+
+@pytest.mark.parametrize("lock_type", [FileLock, SoftFileLock])
+def test_lock_can_be_non_thread_local(
+    tmp_path: Path,
+    lock_type: type[BaseFileLock],
+) -> None:
+    lock = lock_type(tmp_path / "test.lock", thread_local=False)
+
+    for _ in range(2):
+        t = threading.Thread(target=lock.acquire, kwargs={"timeout": 2})
```

Comment:
Don't use one letter variables outside of  comprehension please.

Reply:
fixed